Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -508,6 +512,16 @@
     }
 
     postLayoutData.baseWritingDirection = frame.editor().baseWritingDirectionForSelectionStart();
+
+#if PLATFORM(MAC)
+    if (auto* document = frame.document()) {
+        if (auto* page = document->page()) {
+            const Vector<FloatPoint> offsetsForHitTesting { { -30, 50 }, { 30, 50 }, { -60, 35 }, { 60, 35 }, { 0, 20 } };
+            const auto rects = page->chrome().client().getEvasionRectsAroundSelection(offsetsForHitTesting, false);
```

Comment:
`getEvasionRectsAroundSelection` is on `WebPage`, no need for the indirection.

Reply:
@pxlcoder yes, but `page` is a `Page`, not a `WebPage`

Reply:
Yes, but there's no need to go through `page`, this code is already in `WebPage`.

Reply:
ohhh true, forgot I was in `WebPageCocoa` haha